department-of-veterans-affairs/caseflow
# Intake | Handle submission of Ramp Refiling - PART ONE

The submission logic is a doozy, so I split this up. Apologies on this being a little long, I didn't expect it to be so complicated.

## Testing Instructions
- Attempt to finish a RAMP Refiling with out checking anything, you should see error messages on both the confirmation and the issues selection control.
- Attempt to finish a RAMP refiling with selecting the appropriate options. Validate that they are saved in the DB (it also will create an EP, which can be validated in UAT, but it's probably best to wait till we are also making the contentions)

## Notes
All the AC should be satisfied except:
- creating the contentions, which will be handled in a future PR
- If the selection includes at least one eligible contention, navigates to success confirmation page (#4064)
- If the user only selects the ineligible checkbox, navigate to the ineligible confirmation page (#4206)
- If this is an appeal request, navigate user to "appeal created" confirmation page (#4181)

These will be handled in future PRs

connects #3966
